blog/_posts/2017-05-23-whats-new-scala-native.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: blog
+post-type: blog
+by: Martin Duhem, Guillaume Massé and Denys Shabalin
+title: "What's new with Scala Native?"
+---
+
+It has been a little bit more than two months since Scala Native 0.1 has been
+released. What's new in Scala Native? What should you expect for future
+releases?
+
+## Scala Native 0.2
+
+In April, [Scala Native 0.2 was released][scala-native-0.2-release]. The main
+focus of this release was to increase the coverage of classes from
+the JDK, such that that more programs can be ported to Scala Native without any further
+effort.
+
+### Improvements to the standard library
+
+* Support for file I/O APIs from `java.io` was added by [@Duhemm][@Duhemm]
+  from the Scala Center with help from [@cedricviaccoz][@cedricviaccoz] and
+  [@Korf74][@Korf74] in [#574][#574]. Scala Native now supports enough to read
+  and write files.
+
+  ```scala
+  import java.io.{DataInputStream, File, FileInputStream}
+  val fis = new FileInputStream(new File("hello.txt"))
+  val dis = new DataInputStream(fis)
+  println(dis.readLine())
+  ```
+
+* In [#588][#588], [@MasseGuillaume] from the Scala Center added support for
+  regular expressions. This implementation relies on Google's RE2 engine and
+  [uses a syntax slightly different from the JDK][scala-native-doc-regular-expressions].
+
+  ```scala
+  import java.io.{DataInputStream, File, FileInputStream}
+  val fis = new FileInputStream(new File("hello.txt"))
+  val dis = new DataInputStream(fis)
+  println(dis.readLine())
+  ```
+
+* [@densh][@densh] added initial support for Scala's Futures in [#618][#618],
+  using an implementation similar to that of Scala.js, where Futures will be
+  completed after the `main` method is executed.
+
+  ```scala
+  import java.util.concurrent.Future
+  import scala.concurrent.ExecutionContext.Implicits.global
+
+  object Test {
+    def main(args: Array[String]): Unit = {
+      println("Start")
+      Future {
+        println("Hello from the Future")
+      }.foreach(_ => ())
+      println("End")
+    }
+  }
+  ```
+
+* Scala Native now supports pointer subtraction. This work has been
+  contributed by [@jonas][@jonas] in [#624][#624].
```

Comment:
Maybe explain what pointer subtraction in one sentence for those people who don't know anything about it?